pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -69,7 +83,7 @@ def __init__(
     ):
         super().__init__()
 
-        if shutil.which("nvidia-smi") is None:
+        if shutil.which('nvidia-smi') is None:
```

Comment:
It will not work on Windows

Reply:
Do you consider to use pynvml instead?

Reply:
I can get it to work on my windows machine. Just had to add nvidia-smi to PATH variable, and restart. On my computer it is stored in: `C:\Program Files\NVIDIA Corporation\NVSMI`

Reply:
Yep, but this is something that user should do manually
maybe in that case, also add smth like so?
 ```python
if any([shutil.which('C:/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi') is None,
         shutil.which('nvidia-smi') is None])
```
?

Reply:
That would work in most cases, as long as the user has install in default location.
Maybe it should just be clarified in the docs, that the user needs to be able to call `nvidia-smi` from terminal and hint what windows users should do?

Reply:
or we can add it manually into PATH (os.environ["PATH"]), but i don't know is if a valid behavior